### PR TITLE
nvuApp: added answer to questions

### PR DESCRIPTION
**Pull request by qnislavnachev**
when admin is logged and open some questionnaires
he can see the answer of the questions in it

### CODE REVIEW

**Comment by qnislavnachev** [comment]
admin now can view answers of questions

**Comment by qnislavnachev** [comment]
fixed tests now they are running

File: src/main/java/com/clouway/nvuapp/adapter/http/controllers/ViewQuestionnaireHandler.java [hunk 1]
```diff
@@ -26,9 +27,28 @@ public Response handle(Request request, Tutor tutor) {
     }
 
     private Map<String, Object> getTemplates(Questionnaire questionnaire, int id) {
+        AnsweredQuestions answeredQuestions = new AnsweredQuestions(questionnaire.getQuestions(), questionnaire.getAnswers());
         return new LinkedHashMap<String, Object>(){{
-            put("questions", questionnaire.getQuestions());
+            put("questions", answeredQuestions.getAnsweredQuestion());
```
Comment (mgenov) [inline src/main/java/com/clouway/nvuapp/adapter/http/controllers/ViewQuestionnaireHandler.java]: You don't need a class if you will have only Map of String to Question as simple private method could do the job.

File: src/main/resources/com/clouway/nvuapp/adapter/http/servlet/showTest.html [hunk 2]
```diff
@@ -8,14 +8,15 @@
 <body>
 <#include "/adminNavbar.ftl">
     <h1>Въпросник № ${num}</h1>
-    <#list questions as question>
+    <#list questions as key, value>
```
Comment (mgenov) [inline src/main/resources/com/clouway/nvuapp/adapter/http/servlet/showTest.html]: as answer, question will better describe the intention.

**Comment by qnislavnachev** [comment]
no longer using class to return answered questions and in html key and value are renamed to answer and question

**Pull request merged**